File: Exercicios_2304/Ruby/exercicio6.rb
#custo de fabrica
#distribuidor = +28%
#impostos = +45%
# ? = conmsumidor

puts ("Digite o custo de fábrica")
  custo_fabrica = gets.to_f

distribuidor = (custo_fabrica * 28/100)

impostos = (custo_fabrica * 45/100)

puts ("O custo final do consumidor é R$ #{custo_fabrica + distribuidor + impostos}")

File: Modulo_1/Aula 04 - 23.04.2022/Python/exercicio6.py
custo_fabrica = int(input("Digite o custo de fábrica "))

distribuidora = float(custo_fabrica * (28/100))

impostos = float(custo_fabrica * (45/100))

custo_final = float(custo_fabrica + impostos + distribuidora)

print(f"O custo final para o consumidor é de R$ {custo_final:.2f}")
